theano/sandbox/gpuarray/type.py

```diff
@@ -95,6 +140,21 @@ def __init__(self, dtype, broadcastable, context_name=None, name=None):
                             (self.__class__.__name__, self.dtype))
 
     def clone(self, dtype=None, broadcastable=None):
+        """
```

Comment:
why document in the chield class all the method that are in the parent? I think it should be documented in the parent.